lib/schema.js

```diff
@@ -324,7 +326,7 @@ function PrimitiveSchema(type) {
 
 util.inherits(PrimitiveSchema, Schema);
 
-function FieldSchema(name, type) {
+function FieldSchema(name, type, defaultValue) {
```

Comment:
@blake-nicholson can you let me know if you have a fix for this? At this point I think we should merge the request as-is as I'm not sure how to add a fix for this without breaking the tests that come in.